src/components/payment/payment.module.scss

```diff
@@ -0,0 +1,40 @@
+@import '@styles/theme.module.scss';
+
+.root {
+}
+.boldMessage {
+    font-weight: 600;
+    font-size: 22px;
+    margin-bottom: 10px;
+}
+.title {
+    font-weight: bolder;
+    font-size: 18px;
+    margin-bottom: 10px;
+}
+.subTitle {
+    margin-top: 30px;
+    margin-bottom: 10px;
+    font-weight: 500;
+}
+.btnPayment {
```

Comment:
again, do we really need a new button? maybe to use existing one and combine missing styles with classnames lib?